### ISSUE TITLE
Main navigation missing aria-label

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/vets-design-system-documentation#3812]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

**1.** There are two elements with the role “navigation” – the top nav (beginning with “VA Benefits and Health Care”) and the breadcrumbs – but only the breadcrumbs have a name (`aria-label=”Breadcrumb”`). As a result, if someone is navigating the site by landmark, it may be unclear what that first “navigation” landmark is for.

![Image](https://github.com/user-attachments/assets/5f825419-c499-4a5f-a32d-a87bbac64b44)

---

**2.** Also, for some names of logged-in users, the bottom of the text is cut-off when zoomed.

![Image](https://github.com/user-attachments/assets/0c7081ac-4d62-4f0c-ad41-e0d0fd30799b)

## What I expected to happen

**1.** When a page has more than one "navigation" landmark, I expect each to be labeled so I know what type of navigation it is.

The main nav should have an aria-label:
`<nav aria-label="Main">`

**2.** The text should not get cut off when zoomed.

## Reproducing

- CSS Library version:
- Device: Macbook
- Browser: Chrome

Steps to reproduce:

1. Go to a page with more than one navigation landmark, such as [VA education and training benefits](https://www.va.gov/education/)
2. Turn on VoiceOver
3. Access the rotor menu (VO + U)
4. Arrow to the left until you get to the "Landmarks" menu - note the missing label on the main navigation

## Urgency

How urgent is this request? Please select the appropriate option below and/or provide details

- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

**Comment by rsmithadhoc** [comment]
Hi @coforma-jamie,

Good catch, I think a label would be helpful. However, DST does not manage this header, so I think this bug would need to go to the sitewide team.

Thanks!

**Comment by laflannery** [comment]
@rsmithadhoc (@coforma-jamie  looped me in cause I used to be on Sitewide). The majority of the ownership for the header was transferred to DST last fall, this is [the Product Ownership documentation](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/header-footer#product-ownership).

Based on this, DST would be responsible for this particular update as it seems to fall within the Delivery category

**Comment by rsmithadhoc** [comment]
> [@rsmithadhoc](https://github.com/rsmithadhoc) ([@coforma-jamie](https://github.com/coforma-jamie) looped me in cause I used to be on Sitewide). The majority of the ownership for the header was transferred to DST last fall, this is [the Product Ownership documentation](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/header-footer#product-ownership).
> 
> Based on this, DST would be responsible for this particular update as it seems to fall within the Delivery category

Thanks @laflannery, I was not aware we officially took over on that, just that it was eventually going to happen. I'll let the team know about this issue.

**Comment by rsmithadhoc** [comment]
Edited the issue to include an additional bug with the header reported by Jamie.
